### DISCUSSION TITLE
How to persist nginx.conf changes after update?

### DISCUSSION THREAD

**Discussion by JayP718** [Budibase/budibase#8028] [Help] [answered]
Hi,

Was hoping someone can help me. I made some changes to the nginx.conf file. Is there any way for them to persists after upgrades? Everytime I upgrade budibase I have to redo the changes. 

Thanks,

JP

**Comment by MartinPicc** [comment, marked as answer]
Hi

`nginx.conf` is generated via a template that you can find in bbproxy service `/etc/nginx/templates/nginx.conf.template`.
So you can try to
- launch docker-compose
- open a shell into bbproxy service
- copy the content of this file
- paste it into a local file
- make your modifications into your local file
- bind the local file as a volume in your docker-compose like so:

```
    volumes:
      - /path/to/nginx.conf.template:/etc/nginx/templates/nginx.conf.template
```
And then restart. The changes you did to the template should appear in the `nignx.conf` file

**Reply by JayP718** [reply]
Thanks! I'll try that

**Reply by JayP718** [reply]
That worked thanks again!
